src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,55 @@
+.calendar {
+  width: $calendar-width;
+
+  display: flex;
+  justify-content: flex-start;
+  align-items: flex-end;
+  flex-wrap: wrap;
+  gap: $calendar-gap;
+
+  @include content-padding;
+
+  @for $number from 1 through 31 {
+    & :nth-child(#{$number})::before {
+      content: '#{$number}';
+    }
+  }
+
+
+  &__day {
+    height: $day-block-size;
+    width: $day-block-size;
+    box-sizing: border-box;
+    position: relative;
+    display: none;
+
+    background: $day-default-color;
+    border: 1px solid $day-border-color;
+
+    font-size: 30px;
+  }
+
+  @for $count from 1 through $max-month-length {
+    &-month-length--#{$count} :nth-child(-n + #{$count}) {
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      border: 1px solid $day-border-color;
+      background-color: $day-default-color;
+      transition-duration: $transition-time;
+
+      &:hover {
+        cursor: pointer;
+        background-color: $day-hover-color;
+        transform: translateY(-$day-block-translate);
+      }
+    }
+  }
+
+  @each $coefficient, $name in $days {
```

Comment:
coefficient sounds like a magic number, but it's the number of the column, or order, etc